contracts/seed/SeedFactory.sol

```diff
@@ -44,40 +42,21 @@ contract SeedFactory is CloneFactory {
         _;
     }
 
-    modifier onlyOwner() {
-        require(
-            msg.sender == address(owner),
-            "SeedFactory: protected operation"
-        );
-        _;
-    }
-
     /**
-     * @dev               Initialize proxy.
-     * @param _owner     The address of the owner controlling this contract.
+     * @dev               Initialize SeedFactory.
      * @param _masterCopy The address of the Seed contract which will be a masterCopy for all of the clones.
      */
-    function initialize(Avatar _owner, Seed _masterCopy) external initializer {
-        require(_owner     != Avatar(0), "SeedFactory: owner cannot be null");
+    function initializeMasterCopy(Seed _masterCopy) external initializer onlyOwner {
```

Comment:
I'm curious why we don't just use changeMasterCopy and delete initializeMasterCopy?